### DISCUSSION TITLE
A general note to the version

### DISCUSSION THREAD

**Discussion by mtomas7** [olivierlacan/keep-a-changelog#483] [Ideas]
I read about this project on WP Tavern and I was surprised that the only thing I have to change in my changelog is the format of version title :)

I thought it would be good to include a general Note to the version. Not always it is needed, but sometimes it would be useful. Or you would just include it as a simple paragraph below the version title?

Also I use indicators that group the changes made, like Accessibility, Translation, etc..

It could be like:
### Note
- This changelog reflects theme development going from Tiny Forge v1.6.0 to Tiny Framework v2.0.0
### Added
- Enhancement: new mobile menu look and functionality: sticky menu + integrated search.
- Enhancement: new navigation menu for social networking link management.
- Accessibility: added outline that affects all links on the site.
- Translation: significant changes compared to Tiny Forge, translation will have to be redone.

**Comment by olivierlacan** [comment]
That's not a bad idea at all and I would go so far as to suggest that you don't need a header for this note, which would sufficiently differentiate it from the change entries themselves. 

I think I've seen something similar in quite a few changelogs, especially for major releases.
